runtime/doc/news.txt

```diff
@@ -15,170 +15,129 @@ BREAKING CHANGES                                                *news-breaking*
 
 The following changes may require adaptations in user config or plugins.
 
-• In some cases, the cursor in the Nvim |TUI| used to blink even without
-  configuring 'guicursor' as mentioned in |cursor-blinking|. This was a bug
-  that has now been fixed. If your cursor has stopped blinking, add the
-  following (or similar, adapted to user preference) to your |config| file: >vim
+• API:
+  • |nvim_open_win()| now blocks all autocommands when `noautocmd` is set,
+    rather than just those from setting the `buffer` to display in the window.
 
-    set guicursor+=n-v-c:blinkon500-blinkoff500
-<
-• |vim.islist()| now checks whether a table is actually list-like (i.e.,
-  has integer keys without gaps and starting from 1). For the previous
-  behavior (only check for integer keys, allow gaps or not starting with 1),
-  use |vim.isarray()|.
-
-• "#" followed by a digit no longer stands for a function key at the start of
-  the lhs of a mapping.
-
-• `:behave` was removed.
-  - If you used `:behave xterm`, the following is equivalent: >vim
+• Defaults:
```

Comment:
It the future we should always start with a "skeleton" of various categories, because (1) that helps avoid merge-conflicts and (2) it's helpful for readers.